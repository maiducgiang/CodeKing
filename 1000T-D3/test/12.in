85
84275 6828 57840 52973
37910 88854 28545 27656
14798 47620 22372 8043
98639 37763 59925 49133
89720 27767 63160 18740
21187 70891 26353 45874
70600 28867 38208 4157
36191 80052 62701 31782
93055 86083 2526 67730
9601 7494 82905 76687
75772 55409 23442 12367
72248 8173 12539 32898
17453 37014 11129 46826
44883 44157 95469 15058
57218 27792 77690 72157
14850 50836 87598 65707
51395 74317 18169 77468
1488 43496 12982 18595
53838 37913 26684 53095
72989 99004 41460 72965
13570 20381 53073 64824
32018 85362 67111 51333
17763 69890 32435 16464
90293 97920 89374 75342
56960 68516 59459 65612
67719 42605 46219 43469
4312 98860 74727 65074
71993 98302 18736 4012
19051 22747 34419 27337
87505 60310 89584 53669
28779 31061 43835 64544
34230 20694 69988 24829
69667 46880 1760 54893
68304 54316 16078 44153
8229 78245 26317 50052
17443 90488 86528 78193
55415 22328 78695 78262
18513 58435 1391 58315
47173 5312 66861 32724
14555 20771 80067 3057
15216 23771 48137 40980
48724 33188 49046 27414
78931 18606 68212 15764
63916 64387 74563 93552
5675 20936 30302 90376
62108 37107 30094 42006
99342 70634 69904 95488
86603 43724 19255 6853
94089 14150 84300 49389
3659 68096 77418 18625
24783 62046 83276 95801
88499 72312 89739 32797
48564 5560 18377 12095
50031 73154 27402 88197
81752 27201 9190 56576
89095 23974 64902 8518
63221 79304 25536 44270
84786 32743 24770 16501
83036 68188 93450 57670
82126 1871 73759 37453
51727 28209 88151 55909
33820 64010 17565 2612
73589 64846 47792 54527
93796 47838 21905 83766
51964 87533 93725 95371
75094 4358 23188 31395
54077 81169 79243 65890
90134 25122 30036 64022
73049 91940 83649 20422
93272 48245 15832 97875
34349 62333 58596 65212
35311 70885 51890 93136
8952 69967 10739 94092
95559 85147 2354 7946
25934 18953 86733 33805
10722 28949 69524 73834
56443 57533 85884 20713
42442 66691 15424 16616
43415 31089 93374 77703
20752 7314 75696 75791
53561 34659 50840 1945
59146 11118 69387 65517
55008 61713 31320 11184
22314 9505 15175 48455
45995 89782 50903 6798
