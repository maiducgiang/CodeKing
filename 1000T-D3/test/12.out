4841234124
2502635440
853314098
5503403779
4433305060
2281781950
1714534583
3900350923
452502588
749373213
1162592549
1008398919
606163243
734226600
4291493284
4128050578
2284061328
573017216
2437933435
6404224285
1241141883
5223549178
1983189277
7484510704
4424367724
3012740711
7345887484
781761340
648009811
5673014270
1957543246
1461168894
205122720
1971484360
2254488980
7684358259
4351215401
107034668
2972698209
542191107
1035363715
1671794488
4408193884
6014588805
806366522
2311096736
8751503424
1125275717
7437712050
5056732541
5477349771
3824221857
857532708
3375515370
1001278070
4430693674
3209489158
1829397353
5319947560
5989603108
3650272199
618625190
3953703298
3102492770
8289563169
1777045778
5709984886
3461829216
3072019937
2240497144
3879717452
5134660156
847511141
425381924
1246189238
2058862096
1262717419
1079232848
3566636491
1571538222
1028389435
4060936842
825210072
482853325
2541563671
